keras_cv/models/inception_v3.py

```diff
@@ -0,0 +1,429 @@
+# Copyright 2015 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+"""Inception V3 model for Keras.
+
+Reference:
+  - [Rethinking the Inception Architecture for Computer Vision](
+      http://arxiv.org/abs/1512.00567) (CVPR 2016)
+- [Based on the Original keras.applications Inception V3](https://github.com/keras-team/keras/blob/master/keras/applications/inception_v3.py)
+"""
+
+import tensorflow as tf
+
+from tensorflow.keras import backend
+from tensorflow.keras.applications import imagenet_utils
+from tensorflow.keras.engine import training
+from tensorflow.keras.layers import VersionAwareLayers
+from tensorflow.keras.utils import data_utils
+from tensorflow.keras.utils import layer_utils
+
+# isort: off
+from tensorflow.python.util.tf_export import keras_export
+
+layers = VersionAwareLayers()
+
+def InceptionV3(
```

Comment:
please update the API to match the example set in DenseNet.  there are distinctions between keras.applications and keras_cv.models:

https://github.com/keras-team/keras-cv/blob/master/keras_cv/models/densenet.py